faucet/frontend/scripts/docker/startup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+npm install
```

Comment:
can you run the npm install during the docker build?

Reply:
yes, that makes sense.